lib/client.js

```diff
@@ -9,6 +9,7 @@
 const assert = require('bsert');
 const EventEmitter = require('events');
 const URL = require('url');
+const Path = require('path').posix;
```

Comment:
Did you have to specify `posix` here?

Reply:
Yes, in case it is running on windows. Then it will use windows style path joining with `//` on windows

Reply:
I think we should really use `url.resolve` here
https://nodejs.org/api/url.html#url_url_resolve_from_to

Reply:
That will also resolve `../`, I don't think that's desirable here.